core/modules/node/tests/node.test

```diff
@@ -2088,8 +2088,8 @@ class NodeTypeTestCase extends BackdropWebTestCase {
 
     // Verify that title and body fields are displayed.
     $this->backdropGet('node/add/page');
-    $this->assertRaw('Title', 'Title field was found.');
-    $this->assertRaw('Body', 'Body field was found.');
+    $this->assertRaw('<label for="edit-title">Title', 'Title field was found.');
+    $this->assertRaw('<label for="edit-body-und-0-value">Body', 'Body field was found.');
```

Comment:
These tests were updated because CKEditor 5 puts `Title` in the raw HTML as a JSON string, so we need to be more specific to match only the field label, not `Title` anywhere on the entire page.

Reply:
Is that misleading "Title" the linkTitle from extraAttributes?

Reply:
Yes, that's the place. :+1: